book/website/community-handbook/style/internal-links.md

```diff
@@ -0,0 +1,7 @@
+# Internal Links
+
+To maintain consistency in The Turing Way documentation, please use **relative links** when referencing other parts of the book.
+
+### ✅ Use this format:
+```markdown
+[Contribution Guide](../collaboration/contributing.md)
```

Comment:
I believe, although this works, it isn't the preferred way to handle internal links.

We prefer [labels and cross-references](https://book.the-turing-way.org/community-handbook/style/style-crossref) so that links still work even if files are renamed or sections are moved.